browser/brave_shields/ad_block_pref_service_factory.cc

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_shields/ad_block_pref_service_factory.h"
+#include "brave/components/brave_shields/browser/ad_block_service.h"
+#include "chrome/browser/profiles/incognito_helpers.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/keyed_service/content/browser_context_dependency_manager.h"
+
+namespace brave_shields {
+
+// static
+AdBlockPrefService* AdBlockPrefServiceFactory::GetForBrowserContext(
```

Comment:
Should we just make this `AdBlockServiceFactory` with a comment that it currently only handles prefs until we move the rest of the code?

Reply:
yeah, actually I've put the comment here https://github.com/brave/brave-core/pull/2988/files#diff-0c92a2935eaf3c64ab96614e29705f8eR70